schema.json

```diff
@@ -722,11 +853,34 @@
             "type": "string"
           }
         },
+        "required": [
+          "name",
+          "position",
+          "startDate",
+          "endDate"
```

Comment:
Not sure about `endDate` as not providing it means current as far as I thought. Please correct me, if I missed something.

Reply:
Sorry I will fix it.